src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -374,14 +374,17 @@
       (rf/dispatch [:activity-center.notifications/dismiss-community-overview id]))
     [community-scroll-page community pending?]))
 
-(defn overview
+(defn internal-overview
   [id]
   (let [id                  (or id (rf/sub [:get-screen-params :community-overview]))
         customization-color (rf/sub [:profile/customization-color])]
+    (rn/use-effect #(rf/dispatch [:communities/update-last-opened-at id]) [id])
```

Comment:
This should be part of navigation event which is opening community. (we are adding an event in https://github.com/status-im/status-mobile/pull/18465)

Reply:
Also the `id` of a community never changes, so there's no need to make the dispatch code reactive with `use-effect`.